elyra/pipeline/handlers.py

```diff
@@ -134,3 +156,31 @@ async def get(self, processor, component_id):
         self.set_status(200)
         self.set_header("Content-Type", 'application/json')
         self.finish(ComponentRegistry.to_canvas_properties(component))
+
+
+class PipelineValidationHandler(HttpErrorMixin, APIHandler):
+    """Handler to expose method calls to execute pipelines as batch jobs"""
+
+    @web.authenticated
+    async def get(self):
+        msg_json = dict(title="GET requests are not supported.")
+        self.write(msg_json)
+        self.flush()
+
+    @web.authenticated
+    async def post(self):
+        self.log.debug("Pipeline Validation Handler now executing post request")
+
+        root_dir = self.settings['server_root_dir']
+        expanded_path = get_expanded_path(root_dir)
+
+        pipeline_definition = self.get_json_body()
+        self.log.debug("Pipeline payload: %s", pipeline_definition)
+
+        response = await PipelineValidationManager().validate(pipeline=pipeline_definition,
```

Comment:
The singleton instance should be obtained using `PipelineValidationManager.instance()`.  This comment applies throughout - including tests.